lib/agents/weather/schema.json

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
I believe this file is auto generated by the weather lambda? Is there a cli command to do this?